src/@next/Banner/BannerStyle.ts

```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import * as Breakpoints from '../utilities/breakpoints';
+import { borderRadius8 } from '../utilities/borderRadius';
+import { Blue, Green, Neutral, Orange, Red } from '../utilities/colors';
+
+export const StyledCloseIconWrapper = styled.div`
```

Comment:
why don't we use button instead of div?

Reply:
Shorter styling, will need to remove some natural button attributes.  Do you see any risk with this?

Reply:
i did read that buttons are harder to style vs divs. common risk would be accessibility if we use divs

Reply:
Oh, I added role "button" there to address that